test/peering-test.js

```diff
@@ -0,0 +1,176 @@
+/*!
+ * test/peering-test.js - test for peering
+ * Copyright (c) 2020, Mark Tyneway (MIT License).
+ * https://github.com/handshake-org/hsd
+ */
+
+'use strict';
+
+const assert = require('bsert');
+const {NodeClient, WalletClient} = require('hs-client');
+const FullNode = require('../lib/node/fullnode');
+
+let one, two;
+describe('Peering', function() {
+  afterEach(async () => {
+    if (one)
+      await one.close();
+
+    if (two)
+      await two.close();
+
+    one = null, two = null;
+  });
+
+  it('should create peers (plaintext)', async () => {
```

Comment:
I'd like to deduplicate these two tests using a `for` loop since they share a lot of the same logic. The main difference is what is asserted in the `'peer connect'` event callback function.